5.5.0 release notes (#336)
hazelcast/management-center-docs: docs/modules/release-notes/pages/5-5-0.adoc

= 5.5.0 Release Notes
:description:
These release notes list any new features, enhancements, and fixes that were made between version 5.4.1 and 5.5.0 of Hazelcast Management Center (MC).

> {description}

Reviewer: I don't know why description isn't rendering in the page here so quick fix to replace variable with actual text:

```suggestion
These release notes list any new features, enhancements, and fixes that were made between version 5.4.1 and 5.5.0 of Hazelcast Management Center (MC).
```

> IMPORTANT: If you are upgrading Management Center from a version earlier than 5.4.0, a data migration step is required as Management Center 5.5.0 uses H2 2.x as its internal database instead of H2 1.x (as used in previous releases). If you use the provided `hz-mc` or `mc-start` scripts to start Management Center, this data migration is handled automatically by the script. If you use a custom run script, refer to the documentation for comprehensive instructions on migrating Management Center data. If the internal database is not upgraded, then Management Center will start with a new configuration.

Reviewer: ```suggestion
CAUTION: If you are upgrading Management Center from a version earlier than 5.4.0, a data migration step is required as Management Center 5.5.0 uses H2 2.x as its internal database instead of H2 1.x (as used in previous releases). If you use the provided `hz-mc` or `mc-start` scripts to start Management Center, this data migration is handled automatically by the script. If you use a custom run script, refer to the documentation for comprehensive instructions on migrating Management Center data. If the internal database is not upgraded, then Management Center will start with a new configuration.
```

Reviewer: Looking at other examples, and the coloring applied, we should switch to Caution - sorry for late switch

= 5.5.0 Release Notes
:description:

> These release notes list any new features, enhancements, and fixes that were made between version 5.4.1 and 5.5.0 of Hazelcast Management Center (MC).
> {description}

Reviewer: ```suggestion
These release notes list any new features, enhancements, and fixes that were made between version 5.4.1 and 5.5.0 of Hazelcast Management Center (MC).

{description}
```
Think you need above to make description display.
Bonus question: do we generally us MC as an acronym?

Author: yes, we use MC as an acronym

> 

Reviewer: ```suggestion

NOTE: Hazelcast Management Center 5.5 requires Java 17 or later to run.

```

Author: we had this note in the previous release, there was no upgrade of the minimal required Java version in 5.5. Can we skip it?

Reviewer: Yeah, fine with me - that was my slack DM question tho

> NOTE: This only applies if you are upgrading your MC from a version before 5.4.0: MC 5.5.0 uses H2 2.x as its internal database instead of H2 1.x used by the previous MC releases.
> H2 1.x and 2.x are not compatible so a data migration step is required.
> If you use the provided `hz-mc` or `mc-start` scripts to start MC, no further actions are needed as they handle the migration seamlessly. If you use
> a custom run script, refer to the documentation for comprehensive instructions on migrating MC data. If the internal database is not upgraded, then MC will start with a new configuration.

Reviewer: ```suggestion
IMPORTANT: If you are upgrading Management Center from a version earlier than 5.4.0, a data migration step is required as Management Center 5.5.0 uses H2 2.x as its internal database instead of H2 1.x (as used in previous releases). If you use the provided `hz-mc` or `mc-start` scripts to start Management Center, this data migration is handled automatically by the script. If you use a custom run script, refer to the documentation for comprehensive instructions on migrating Management Center data. If the internal database is not upgraded, then Management Center will start with a new configuration.
```
What is H2?
"refer to the documentation" which documentation? Is it ours? Can we link?

Author: H2 is a database as mentioned at the beginning. The documentation is ours https://docs.hazelcast.com/management-center/5.4/deploy-manage/h2-upgrade

== New Features

* Added screens for Vector Collection data structures. [MC-2875]
* Added support for multi-version Rolling Upgrade. [MC-2820]
* Added support for MULTI_MEMBER clients. [MC-2638]
* Added support for multi-resource user code namespaces. [MC-2772]
* Added support for Log4j2 JsonTemplateLayout logging. [MC-2844]
* Added an ability to demote a data member to a lite member. [MC-2823]
* Added an ability to preview Client Filtering results. [MC-2715]
* Added an ability to specify allowed Client Filtering modes. [MC-2715]

== Enhancements
* Improved logging of H2 Database upgrade process. [MC-2668]
* Improved styling of WAN Replication table. [MC-2711]
* HealthCheck icon now reflects the highest severity issue detected. [MC-2867]
* Added cpDirectToLeader property value to Client screen. [MC-2902]
* Added Slow Operations details to Member Listing screen. [MC-2783]
* Added a link to the License Feature CP Section if CP Subsystem is disabled. [MC-2847]

> * Added an ability to a WAN Replication reference of an IMaps. [MC-2729]
> * Added an ability to configure `consistency-check-strategy` during dynamic creation of a WAN publisher. [MC-243]
> * Added an ability to load OIDC configuration parameters from an OIDC configuration endpoint. [MC-2733]

Reviewer: ```suggestion
* Added ability for WAN Replication reference of an IMaps. [MC-2729]
* Added ability to configure `consistency-check-strategy` during dynamic creation of a WAN publisher. [MC-243]
* Added ability to load OIDC configuration parameters from an OIDC configuration endpoint. [MC-2733]
```

NOTE: in Clustered REST API `memberConnection` allowed field values have been changed from `ALL` | `<member-address>` to
`ALL_MEMBERS` | `MULTI_MEMBER` | `SINGLE_MEMBER`

== Fixes

* Fixed layout on Members Listing screen when slow operations are detected. [MC-2918]
* Fixed links in license expiration popups. [MC-2719]

> * Jet job Received Items count no longer increases the stream has been cancelled. [MC-2761]

Reviewer: ```suggestion
* Jet job Received Items count no longer increases if the stream has been cancelled. [MC-2761]
```
